### ISSUE TITLE
bug: IE11 Focus ring does not appear on all sides in Select Mobile Mode

### ISSUE THREAD

**Issue by kavya-b** [SAP/fundamental-styles#762]
**Describe the bug**

> Is this issue related to a specific component?
Yes, Select using List component

> What version of the Fundamental Library Styles are you using?
0.7.0

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
Angular

**To Reproduce**
1. Go to 'https://sap.github.io/fundamental-styles/components/select.html' in IE
2. Scroll down to 'Select Mobile Mode'
3. click on any of the items
4. See error: the left and right side of focus ring is missing

**Expected behavior**
Focus ring should be visible in all sides. Ideally it should be an inner focus, but IE11 does not support outline-offset property which is why inner focus anyway does not work. But focus border should at least appear on all sides.

**Screenshots**
![image](https://user-images.githubusercontent.com/53509521/75979894-788ae600-5f07-11ea-8a5a-b9e184b052dc.png)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows
 - Browser [e.g. chrome, safari] IE
 - Version [e.g. 22] 11

**Comment by droshev** [comment]
@JKMarkowski can you check whether we can reproduce it?

**Comment by InnaAtanasova** [comment]
I checked and I could reproduce the issue
